src/modules/footer-focus.js

```diff
@@ -3,7 +3,7 @@ export function getFooterElements() {
 		{
 			id: 'footercredit',
 			selector: 'a[data-type="footer-credit"]',
-			type: 'footer',
+			type: 'footerCredit',
```

Comment:
I think we should probably be explicit about the title, since we have these objects anyway.

Something like:

```
{
  id: 'footercredit',
  selector: '...',
  type: 'footerCredit',
  position: 'middle',
  title: 'footer credit',
}
```